<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(50, 50, 50);
color: #b1b1b1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1,1,6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Алфавит</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_alphabet">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="placeholderText">
           <string notr="true" comment="abcd" extracomment="abc"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Кратность: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Фиксированная подцепочка: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_child_word"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Минимальная длина цепочки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_min_len"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Максимальная длина цепочки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_max_len"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Длина рекурсии</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_step_count"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbtn_ll">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>ЛЛ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_pl">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>ПЛ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_generate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 10px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}
</string>
      </property>
      <property name="text">
       <string>Генерировать</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>583</width>
           <height>360</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="log">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="btn_aftor"/>
    <addaction name="btn_tema"/>
    <addaction name="btn_export"/>
    <addaction name="btn_import"/>
    <addaction name="btn_about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="btn_aftor">
   <property name="text">
    <string>Автор</string>
   </property>
  </action>
  <action name="btn_tema">
   <property name="text">
    <string>Тема</string>
   </property>
  </action>
  <action name="btn_export">
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="btn_import">
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="btn_about">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
